packages/cos/setup.yaml

```diff
@@ -25,49 +25,3 @@ stages:
         DOCUMENTATION_URL: "https://github.com/mudler/cOS"
         VERSION: ":VERSION:"
         PRETTY_NAME: ":PRETTY_NAME:"
-    - name: "Grub"
-      directories:
-      - path: /etc/cos
-        permissions: 0755
-        owner: 0
-        group: 0
-      files:
-      - path: /etc/cos/grub.cfg
```

Comment:
Note: this is being used here currently when refreshing grub during `cos-reset`. https://github.com/rancher-sandbox/cOS-toolkit/blob/master/packages/installer/reset.sh#L71

Reply:
do we have to carry the same for bootargs?

Reply:
I think I've misread it, it should be ok as the grub package ships in /etc/cos/grub.cfg at the end

Reply:
No bootargs are not _installed_ they are just directly read from the loop device at runtime from `/etc/cos/bootargs.cfg`. See https://github.com/rancher-sandbox/cOS-toolkit/blob/481d882cf9f964dcd7113695026e14ee25cdb505/packages/grub-config/config/grub.cfg#L13-L17

Reply:
So each image is free to provide its own bootargs config and obligated to provide a valid ones, otherwise it won't boot.